sw/device/lib/arch/device_fpga_nexysvideo.c

```diff
@@ -13,7 +13,7 @@ const device_type_t kDeviceType = kDeviceFpgaNexysVideo;
 
 const uint64_t kClockFreqCpuHz = 50 * 1000 * 1000;  // 50MHz
 
-const uint64_t kClockFreqPeripheralHz = 50 * 1000 * 1000;  // 50MHz
+const uint64_t kClockFreqPeripheralHz = 12.5 * 1000 * 1000;  // 12.5MHz
```

Comment:
I'm not sure that this will do what you want. Better just to write this as `125 * 100 * 1000"?

Reply:
ah will this actually be turned into 12MHz?